doc/README.md

```diff
@@ -23,7 +23,7 @@ Unpack the files into a directory and run:
 
 Unpack the files into a directory, and then run bitcoin-qt.exe.
 
-### OS X
+### macOS
```

Comment:
thanks for doing that. I have only one doubt and it related the substitution of "OS X" string with "macOS".

I'm a bit out of the Apple world loop so I don't know if this is correct or not, wonder if someone with more up-to-date knowledge could chime in.

Reply:
After Mac OS X 10.11 El Capitan, Apple rebranded it to macOS, starting with macOS 10.12 Sierra. The reason was to make the entire OS lineup across devices use the same naming. macOS, iOS, tvOS, watchOS.